tests/console/sudo.pm

```diff
@@ -91,6 +91,9 @@ sub run {
     sudo_with_pw 'sudo env', grep => '-v ENVVAR=test132', env => 'ENVVAR test132';
     # sudoers configuration
     test_sudoers;
+    become_root;
+    assert_script_run 'test -f /etc/sudoers || cp /usr/etc/sudoers /etc/sudoers';
```

Comment:
Can be done after the `zypper_call 'in sudo expect';` above

Reply:
it could - but I wanted to have all the etc/sudoers.d/* addon tests done with the default config before copying the sudoers to /etc

Reply:
`become_root` uses `sudo`, so IMO a bit weird

Reply:
no, it uses `su -c` - see https://openqa.opensuse.org/tests/3764849#step/sudo/79

Reply:
Great, so `susedistribution::script_sudo` actually calls `su`...

Reply:
but I see where the confusion comes from: `become_root` calls `script_sudo` - which in turn does call `su -c` (not sudo as the name would imply)